src/resources/user/user.controller.ts

```diff
@@ -1,19 +1,21 @@
-export const getMany = (req, res) => {
-	res.json('GET ALL USER')
-}
+import {findAllUser, findUserWithId} from '../../mockDB/db'
 
-export const getOne = (req, res) => {
-	res.json(`GET USER WITH ID ${req.params.id}`)
+export const getMany = (req, res, next) => {
+	findAllUser()
+		.then(users => res.status(200).json(users))
+		.catch(next)
 }
 
-export const createOne = (req, res) => {
-	res.json('CREATE USER')
```

Comment:
Why not using `createUser` for `createOne` but delete it?

Reply:
createUser in this case is actually register (User needs username and password), therefore it should go to /auth/register.